pages/common/routers/datatable.js

```diff
@@ -41,16 +41,19 @@ module.exports = ({
   apiPath,
   schema,
   defaultRowCount = 10
-}) => {
+} = {}) => {
   const app = Router();
 
   const _configure = (req, res, next) => {
     req.datatable = req.datatable || {};
+    req.datatable.schema = schema;
     return configure(req, res, next);
   };
 
   const _getApiPath = (req, res, next) => {
-    req.datatable.apiPath = apiPath;
+    if (apiPath) {
+      req.datatable.apiPath = apiPath;
+    }
```

Comment:
What's the context where an `apiPath` would _not_ be set?

Reply:
`apiPath` was coming through as `undefined` and overwriting the path we actually wanted to use in some cases.

It has something to do with the way we're dynamically assigning the schema / api path in configure here:
https://github.com/UKHomeOffice/asl-internal-ui/pull/40/files#diff-8fcd7a31470eb38e82be8f8cc7f5a094R27

I can't remember the exact reason it was happening because @joefitter was helping with a few issues around the search at the same time.

Reply:
That's happening because the datatable router is expecting to do [this work](https://github.com/UKHomeOffice/asl-internal-ui/blob/0693623f74e441d51861dfa6939a992ca0b11924/pages/search/index.js#L37-L49) in the `getApiPath` method, rather than in configure. In fact that method exists exactly for this purpose.

Reply:
Sorry, not sorry